lib/network_utils.ml

```diff
@@ -246,7 +246,7 @@ module Sysfs = struct
     try
       Sys.readdir ("/sys/bus/pci/drivers/" ^ driver)
       |> Array.to_list
-      |> List.filter (Re.execp (Re.Perl.compile_pat "\\d+:\\d+:\\d+\\.\\d+"))
+      |> List.filter (Re.execp (Re.Perl.compile_pat "\\d+:[a-f\\d]+:[a-f\\d]+\\.\\d+"))
```

Comment:
What about uppercase hex letters `A` to `F`? To avoid confusion with backslah-escaping, this could be also written as:

```
{|\d+:[a-f\d]+:[a-f\d]+\.\d|}
```

Reply:
Hi @lindig 
It's good to know this kind of regex string. Thanks!
And for the upper cases, the name of files under `/sys/bus/pci/drivers/` are always in lower cases. So I think it is not necessary to add matching for A-F.